### ISSUE TITLE
mkdocs serve prints duplicate log message

### ISSUE THREAD

**Issue by rveguilla** [mkdocs/mkdocs#2157]
I'm seeing duplicate log messages when running `mkdocs serve` using version 1.1.2.

Steps to reproduce:
```
$ mkdocs -V
mkdocs, version 1.1.2 from /home/rveguilla/.pyenv/versions/3.8.3/lib/python3.8/site-packages/mkdocs (Python 3.8)
 $ mkdocs new test-mkdocs
INFO    -  Creating project directory: test-mkdocs 
INFO    -  Writing config file: test-mkdocs/mkdocs.yml 
INFO    -  Writing initial docs: test-mkdocs/docs/index.md 
 $ cd test-mkdocs
 $ mkdocs serve
INFO    -  Building documentation... 
INFO    -  Cleaning site directory 
INFO    -  Documentation built in 0.04 seconds 
[I 200721 13:09:15 server:334] Serving on http://127.0.0.1:8000
INFO    -  Serving on http://127.0.0.1:8000
[I 200721 13:09:15 handlers:62] Start watching changes
INFO    -  Start watching changes
[I 200721 13:09:15 handlers:64] Start detecting changes
INFO    -  Start detecting changes
```

**Comment by jimenezk** [comment]
Me too. Not sure if this is why it takes long to build every time.

**Comment by ktomk** [comment]
Hasn't it been always this way? AFAIK I'm seeing them since I started with mkdocs. For me this looks like not as if things are doubled (by doing things - apart from logging itself - two times each) but that there are two loggers (so just two times the log messags in a different presentation):

~~~
[I 200721 13:09:15 server:334] Serving on http://127.0.0.1:8000
~~~
prefixed, tech-bla, certainly for a log-service

~~~
INFO    -  Serving on http://127.0.0.1:8000
~~~
human readable process logging

looking further, when invoked with `-q, --quiet`  the logging starts with serving (and not with building):

~~~
mkdocs serve -q    
[I 210522 17:14:04 server:335] Serving on http://127.0.0.1:8000
INFO    -  Serving on http://127.0.0.1:8000
[I 210522 17:14:04 handlers:62] Start watching changes
INFO    -  Start watching changes
[I 210522 17:14:04 handlers:64] Start detecting changes
INFO    -  Start detecting changes
~~~

the log output is on standard error (stderr / handle #2).

**Comment by waylan** [comment]
Yes, @ktomk you are correct. Regardless, this will be fixed in #2385.

**Comment by ktomk** [comment]
Yes @waylan, now being at 133193a and it does not have this issue any longer - _much_ better by the way.

**Comment by waylan** [comment]
Closing this as fixed in #2385.
